pytorch_lightning/trainer/training_loop.py

```diff
@@ -515,8 +516,22 @@ def run_training_batch(self, batch, batch_idx):
         for split_idx, split_batch in enumerate(splits):
             self.split_idx = split_idx
 
+            def get_optimizers_iterable():
```

Comment:
this method starts to be quite heavy, could we get the function outside?

Reply:
I didn't want `get_optimizers_iterable()` to be part of the Trainer public API.
We can prefix its name with a _
Where do you suggest to put it?

Reply:
_ prefix sounds good

Reply:
Although, there is no other _method() in this file and it doesn't seem to follow the convention.
for example, see `optimizer_closure`.

Reply:
The convention is not fixed yet everywhere lol